### PR TITLE
Source Facebook Pages: parse datetime to rfc3339

### PR DESCRIPTION
**Pull request by artem1205**
## What
Resolving https://github.com/airbytehq/oncall/issues/1476

## How
*Describe the solution*

## Recommended reading order
1. `x.java`
2. `y.python`

## 🚨 User Impact 🚨
Are there any breaking changes? What is the end result perceived by the user? If yes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub and connector version bumped by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)

</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-facebook-pages/source_facebook_pages/components.py [hunk 1]
```diff
@@ -51,49 +51,48 @@ def generate_page_access_token(self) -> str:
...
-    ```
-    """
+    def _date_time_to_rfc3339(self, record: MutableMapping[str, Any]) -> MutableMapping[str, Any]:
+        """
+        Transform 'date-time' items to RFC3339 format
+        """
+        date_time_paths = self._get_date_time_dpath_from_schema()
+        for path in date_time_paths:
+            if "*" not in path:
+                if field_value := dpath.util.get(record, path, default=None):
```
Comment (bazarnov) [inline airbyte-integrations/connectors/source-facebook-pages/source_facebook_pages/components.py:83]: Nicely done here

**Review by bazarnov — APPROVED** [review]
LGTM, for the solution, the unit test should be corrected still.

**Review by clnoll — APPROVED** [review]
LGTM @artem1205!

File: airbyte-config/init/src/main/resources/seed/source_definitions.yaml [hunk 2]
```diff
@@ -514,11 +514,14 @@
...
   dockerRepository: airbyte/source-facebook-pages
-  dockerImageTag: 0.2.2
+  dockerImageTag: 0.2.3
   documentationUrl: https://docs.airbyte.com/integrations/sources/facebook-pages
   icon: facebook.svg
   sourceType: api
   releaseStage: beta
+  allowedHosts:
+    hosts:
+      - "graph.facebook.com"
```
Comment (evantahler) [inline airbyte-config/init/src/main/resources/seed/source_definitions.yaml:524]: 👍 allowed hosts!

File: airbyte-integrations/connectors/source-facebook-pages/source_facebook_pages/schemas/post.json [hunk 3]
```diff
@@ -22,7 +22,7 @@
     },
     "backdated_time": {
       "type": ["string", "null"],
-      "format": "datetime"
+      "format": "date-time"
```
Comment (evantahler) [inline airbyte-integrations/connectors/source-facebook-pages/source_facebook_pages/schemas/post.json:25]: Yup- it's date-time! https://docs.airbyte.com/understanding-airbyte/supported-data-types#specific-types

File: connectors.md [hunk 4]
```diff
@@ -68,7 +68,7 @@
...
-| **Facebook Pages** | <img alt="Facebook Pages icon" src="https://raw.githubusercontent.com/airbytehq/airbyte/master/airbyte-config/init/src/main/resources/icons/facebook.svg" height="30" height="30"/> | Source | airbyte/source-facebook-pages:0.2.2 | beta | [link](https://docs.airbyte.com/integrations/sources/facebook-pages) | [code](https://github.com/airbytehq/airbyte/tree/master/airbyte-integrations/connectors/source-facebook-pages) | <small>`010eb12f-837b-4685-892d-0a39f76a98f5`</small> |
+| **Facebook Pages** | <img alt="Facebook Pages icon" src="https://raw.githubusercontent.com/airbytehq/airbyte/master/airbyte-config/init/src/main/resources/icons/facebook.svg" height="30" height="30"/> | Source | airbyte/source-facebook-pages:0.2.3 | beta | [link](https://docs.airbyte.com/integrations/sources/facebook-pages) | [code](https://github.com/airbytehq/airbyte/tree/master/airbyte-integrations/connectors/source-facebook-pages) | <small>`010eb12f-837b-4685-892d-0a39f76a98f5`</small> |
```
Comment (evantahler) [inline connectors.md:71]: hooray!

**Review by evantahler — APPROVED** [review]
👍

**Pull request merged**